packages/smooth_app/lib/pages/preferences/user_preferences_contribute.dart

```diff
@@ -108,6 +108,12 @@ class UserPreferencesContribute extends AbstractUserPreferences {
               UserPreferencesListTile.getTintedIcon(Icons.open_in_new, context),
           externalLink: true,
         ),
+        _getListTile(
+            'Help improve Open Food Facts in your country',
+            () async => LaunchUrlHelper.launchURL(
+                'https://wiki.openfoodfacts.org/Country_Support_- ${userPreferences.userCountryCode ?? 'France'}'),
+            const IconData(0xf68d),
```

Comment:
There's a globe icon you should reuse: the one used in the app country selector.

Reply:
You're right ill look